packages/ilp-packet/package.json

```diff
@@ -25,10 +25,12 @@
     "codecov": "curl -s https://codecov.io/bash | bash -s - -s coverage -F ilp_packet"
   },
   "dependencies": {
-    "oer-utils": "^5.1.3-alpha.2"
+    "oer-utils": "^5.1.3-alpha.2",
+    "long": "^4.0.0"
```

Comment:
ilp-packet uses long, but didn't declare it before, so when upgrading to webpack 5, which seems to be stricter somehow, it would complain